HLTrigger/HLTcore/src/HLTPrescaleProvider.cc

```diff
@@ -363,11 +363,11 @@ void HLTPrescaleProvider::checkL1TGlobalUtil() const {
 }
 
 template <>
-unsigned int HLTPrescaleProvider::prescaleValue<unsigned int>(const edm::Event& iEvent,
-                                                              const edm::EventSetup& iSetup,
-                                                              const std::string& trigger) {
+double HLTPrescaleProvider::prescaleValue<double>(const edm::Event& iEvent,
+                                                  const edm::EventSetup& iSetup,
+                                                  const std::string& trigger) {
```

Comment:
Given the comment here: https://github.com/cms-sw/cmssw/blob/master/HLTrigger/HLTcore/interface/HLTPrescaleProvider.h#L63-L69
this is the mentioned template specialisation for unsigned int, thus it should NOT be changed to float!
See above!